.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,30 @@
+name: Unit Tests
```

Comment:
Rename this to "Python unit tests", since we have JS unit tests as well.

Reply:
Done